tests/api/test_literature_covers.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# CDS Books is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from mock import MagicMock
+
+from cds_books.literatures.covers import build_literatures_cover_urls
+from cds_books.literatures.tasks import is_allowed, select_cover, update_cover
+
+
+def test_cover_url_builder(app, testdata):
+    """Test builder for cover urls."""
+    doc = testdata["documents"][0]
+    isbn = doc["cover_metadata"]["isbn"]
+    cover_urls = build_literatures_cover_urls(doc)
+    assert cover_urls["small"].endswith("&isbn={}/SC.gif".format(isbn))
+    assert cover_urls["medium"].endswith("&isbn={}/MC.gif".format(isbn))
+    assert cover_urls["large"].endswith("&isbn={}/LC.gif".format(isbn))
+
+    serie = testdata["series"][0]
+    issn = serie["cover_metadata"]["issn"]
+    cover_urls = build_literatures_cover_urls(serie)
+    assert cover_urls["small"].endswith("&issn={}/SC.gif".format(issn))
+    assert cover_urls["medium"].endswith("&issn={}/MC.gif".format(issn))
+    assert cover_urls["large"].endswith("&issn={}/LC.gif".format(issn))
+
+
+def test_is_allowed(app, testdata):
+    """Test is_allowed helper function."""
+    doc = testdata["documents"][0]
+    assert is_allowed(doc)
+
+    series = testdata["series"][0]
+    assert is_allowed(series)
+
+    item = testdata["items"][0]
+    assert not is_allowed(item)
+
+
+def test_update_cover(app, testdata):
+    """Test to update cover_metadata value."""
+    doc = testdata["documents"][1]
+    update_cover(app, record=doc)
+    assert doc["cover_metadata"]["isbn"] == "0123456789"
+
+    serie = testdata["series"][1]
+    update_cover(app, record=serie)
+    assert serie["cover_metadata"]["issn"] == "0123456789"
+
+
+class MockResponse(object):
+
+    def __init__(self, resp_data, code=200, msg='OK'):
+        self.resp_data = resp_data
+        self.code = code
+        self.msg = msg
+        self.headers = {
+            'Content-Length': 1024
+        }
+
+    def get(self, header):
+        return self.headers[header]
+
+    def read(self):
+        return self.resp_data
+
+    def getcode(self):
+        return self.code
+
+
+def test_select_cover_task(app, testdata):
+    """Test to update cover_metadata."""
+    app.config.update(CELERY_TASK_ALWAYS_EAGER=True)
```

Comment:
this should be already in the `conftest.py`, otherwise we should add it there and not here.